pip/compat.py

```diff
@@ -20,6 +20,12 @@
         ipaddress.ip_address = ipaddress.IPAddress
         ipaddress.ip_network = ipaddress.IPNetwork
 
+__all__ = [
+    "ipaddress", "uses_pycache", "console_to_str", "native_str",
+    "get_path_uid", "stdlib_pkgs", "WINDOWS", "samefile",
+    "ThreadPoolExecutor", "ProcessPoolExecutor",
+]
+
 try:
```

Comment:
@pradyunsg Why was this not a valid idea? Is there a circular import or something?

Reply:
Not really sure...